torch/_functorch/config.py

```diff
@@ -374,6 +374,13 @@ def remote_autograd_cache_default() -> Optional[bool]:
 # This callback is invoked on the joint graph before partitioning
 joint_custom_pass: Callable = None  # type: ignore[assignment]
 
+# Note [Selective Decomposition]
+# This config allows for selective decomposition of certain operators in the graph.
+# The current use case is regional inductor compile, where users specify nodes that
+# should be decomposed and compiled with inductor. When true, we skip decomposition
+# for all operators except the user specified nodes.
+selective_decompose: bool = False
```

Comment:
TBH I don't understand what this config does. In particular, if this is "selective" decompose, how exactly do I specify if something is to be decomposed or not? The documentation here doesn't say.